### ISSUE TITLE
Technical/Network

### ISSUE THREAD

**Issue by elsantner** [elsantner/Risk#6]
Das Gerät von einem Nutzer muss der Server sein --> würde Sinn machen, dass wir das Gerät nehmen, wo das Game gestartet wird.

Wird kein Matchmaking Server benötigt? Wie findet ein Geräte sonst andere, die das Spiel installiert haben?

**Comment by WilhelmStu** [comment]
Man könnte das Matchmaking auf das lokale Netzwerk begrenzen, sodass nur Spieler des gleichen Netzwerkes füreinander sichtbar sind. 
Ein Gerät ist der Host, welcher folgend bei der lokalen Spielsuche für alle anderen Teilnehmer angezeigt wird.

**Comment by cusAAU** [comment]
> Man könnte das Matchmaking auf das lokale Netzwerk begrenzen, sodass nur Spieler des gleichen Netzwerkes füreinander sichtbar sind.
> Ein Gerät ist der Host, welcher folgend bei der lokalen Spielsuche für alle anderen Teilnehmer angezeigt wird.

ja, das minimiert bestimmt die Latenz, wenn es im. gleichen Netzt gespielt wird. 
Ich bin mir nur nicht sicher, ob wir das beschränken sollten.. Vielleicht die Spiele + Spielern im gleichen Netz als erstes anzeigen und dann den Rest?

**Comment by elsantner** [comment]
> > Man könnte das Matchmaking auf das lokale Netzwerk begrenzen, sodass nur Spieler des gleichen Netzwerkes füreinander sichtbar sind.
> > Ein Gerät ist der Host, welcher folgend bei der lokalen Spielsuche für alle anderen Teilnehmer angezeigt wird.
> 
> ja, das minimiert bestimmt die Latenz, wenn es im. gleichen Netzt gespielt wird.
> Ich bin mir nur nicht sicher, ob wir das beschränken sollten.. Vielleicht die Spiele + Spielern im gleichen Netz als erstes anzeigen und dann den Rest?

Ja, so sehe ich das auch.
Aber bräuchten wir dafür nicht einen dedicated Server, der die Spieler vermittelt? Wie sonst sollen sich die Endgeräte untereinander netzwerkübergreifend finden?

**Comment by cusAAU** [comment]
> > ja, das minimiert bestimmt die Latenz, wenn es im. gleichen Netzt gespielt wird.
> > Ich bin mir nur nicht sicher, ob wir das beschränken sollten.. Vielleicht die Spiele + Spielern im gleichen Netz als erstes anzeigen und dann den Rest?
> 
> Ja, so sehe ich das auch.
> Aber bräuchten wir dafür nicht einen dedicated Server, der die Spieler vermittelt? Wie sonst sollen sich die Endgeräte untereinander netzwerkübergreifend finden?

Oops, ja :D ich sehe momentan nicht, wie wir den Server aus dem besprochenen Setup weglassen können.
